react/components/SKUSelector/components/SelectorItem.js

```diff
@@ -19,15 +19,16 @@ const getDiscount = (maxPrice, price) => {
 const SelectorItem = ({
   isAvailable = true,
   isSelected = false,
-  children = null,
   maxPrice,
   price,
   onClick,
   isImage,
   variationValue,
+  imageUrl,
+  imageLabel,
+  isFaded,
```

Comment:
⛏ I think this name should be semantic, because the styling might change, but the meaning should not. something like, `isImpossible` maybe? (since that's the term that was used)